.maintain/gitlab/check_line_width.sh

```diff
@@ -5,19 +5,20 @@
 set -e
 set -o pipefail
 
-BASE_ORIGIN="origin"
-BASE_BRANCH_NAME="master"
 LINE_WIDTH="120"
 GOOD_LINE_WIDTH="100"
-BASE_BRANCH="${BASE_ORIGIN}/${BASE_BRANCH_NAME}"
 
-git fetch ${BASE_ORIGIN} ${BASE_BRANCH_NAME} --depth 1
```

Comment:
this is definitely required as the master branch to compare with is not always available.

Reply:
Not if `$CI_COMMIT_BEFORE_SHA` would be working as that commit would be part of the history of the current branch, too.

Reply:
not if the pull request contains more than 100 commits:
https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L35

(that limit had been raised because of such scenarios, but it's safer to just make sure what is compared with is present)